delivery_send_to_shipper_at_operation/models/stock_picking.py

```diff
@@ -53,7 +53,10 @@ def _handle_send_to_shipper_at_operation(self):
                 self.delivery_notification_sent = True
                 related_ship.delivery_notification_sent = True
                 related_ship.carrier_price = self.carrier_price
-                related_ship.carrier_tracking_ref = self.carrier_tracking_ref
+                if not related_ship.carrier_tracking_ref:
+                    related_ship.carrier_tracking_ref = self.carrier_tracking_ref
+                else:
+                    related_ship.carrier_tracking_ref += "," + self.carrier_tracking_ref
```

Comment:
I would start with the positive check, easier to read:
```suggestion
                if related_ship.carrier_tracking_ref:
                    related_ship.carrier_tracking_ref += "," + self.carrier_tracking_ref
                else:
                    related_ship.carrier_tracking_ref = self.carrier_tracking_ref
```